lib/tasks/livesplit_core.rake

```diff
@@ -50,5 +50,10 @@ task :update_lsc do
   mv("#{DEST_FOLDER}/liblivesplit_core.so", "#{PARSER_FOLDER}/liblivesplit_core.so")
   mv("#{DEST_FOLDER}/bindings/LiveSplitCore.rb", "#{PARSER_FOLDER}/LiveSplitCore.rb")
   rm_rf DEST_FOLDER
+  data = File.read("#{PARSER_FOLDER}/LiveSplitCore.rb")
+  data.sub!('module LiveSplitCore', 'module Parser::LiveSplitCore')
+  File.open("#{PARSER_FOLDER}/LiveSplitCore.rb", 'w') do |f|
+    f.write(data)
+  end
```

Comment:
Another possible solution to this problem in case you haven't seen it, though I have no opinion either way:

https://stackoverflow.com/a/8762740

Reply:
Thanks for the link, I ended up liking that way a bit more 👍